src/main/java/fr/openent/homeworkAssistance/models/FeaturedResources.java

```diff
@@ -0,0 +1,70 @@
+package fr.openent.homeworkAssistance.models;
+
+public class FeaturedResources {
+    private String idRessource;
+    private String nomRessource;
+    private String urlVignette;
+    private String urlAccesRessource;
+    private String description;
+
+    public FeaturedResources(String idRessource, String nomRessource, String urlVignette, String urlAccesRessource, String description) {
+        this.idRessource = idRessource;
+        this.nomRessource = nomRessource;
+        this.urlVignette = urlVignette;
+        this.urlAccesRessource = urlAccesRessource;
+        this.description = description;
+    }
```

Comment:
fais plutot un constructeur de ce genre :
```suggestion
    public FeaturedResources(JsonObject featuredResources) {
        setIdResource(featuredResources.getString(ID_RESOURCE, null);
        ...
    }
```